book/website/profiles.yml

```diff
@@ -119,23 +177,20 @@
     - collaboration/organising-conference
     - collaboration/chairing
     - collaboration/event-tools
-    - project-design/project-design
-    - communication/communication
-    - communication/comms-overview
-    - communication/os-comms
-    - communication/comms-overview/comms-overview-principles
-    - communication/dif-articles
-    - communication/presentations
-    - communication/social-media
     - collaboration/collaboration
-    - collaboration/github-novice/github-novice-advanced
     - collaboration/shared-ownership
-    - collaboration/oss-sustainability/oss-sustainability-challenges
+    - collaboration/team-manual
+    - collaboration/stakeholder-engagement
     - ethical-research/ethics-intro
+    - ethical-research/law-policy
+    - ethical-research/internal-policy
+    - ethical-research/data-hazards
+    - ethical-research/ethics-open-source-governance
```

Comment:
Can we add this chapter on Culture change: https://deploy-preview-4043--the-turing-way.netlify.app/ethical-research/cultural-change/cc-examples?

Reply:
This too: https://deploy-preview-4043--the-turing-way.netlify.app/ethical-research/cultural-change/alt-models-discussion

Reply:
Can you add the culture change

Reply:
Added.